SuperPong/ProjectSettings/ProjectVersion.txt

```diff
@@ -0,0 +1 @@
+m_EditorVersion: 5.5.0f3
```

Comment:
When I checked out the project it complained that I was opening an older project version. It still opened and worked fine. Austin and I were talking and we are going to try adding it to the .gitignore